src/status_im/ui/screens/chat/components/reply.cljs

```diff
@@ -81,15 +81,16 @@
        [icons/icon :main-icons/close-circle {:container-style (styles/close-button)
                                              :color (:icon-02 @quo.colors/theme)}]]]]))
 
-(defn reply-message [{:keys [from identicon content-type contentType parsed-text content]} in-chat-input?]
+; This component is also used for quoted pinned message as the UI is very similar
```

Comment:
Is this really needed?

Reply:
What do you mean exactly?

Reply:
I personally find comments like this unnecessary. It is pretty easy to read this from the code and see that this is the case. Comments can often go out of date when code gets updated and mislead developers. Code is the source of truth. There's nothing in the code that really needs a comment to explain here.

Just my opinion 👍

Reply:
That's right. Here the component name is `reply-message` which I also use for the new system pin message, so thought would add a comment to make it clearer.